src/parser/parser.yy

```diff
@@ -46,6 +46,7 @@ class GraphScanner;
     nebula::VertexIDList                   *vid_list;
     nebula::OverClause                     *over_clause;
     nebula::WhereClause                    *where_clause;
+    nebula::WhereClause                    *when_clause;
```

Comment:
WhenClause

Reply:
Do we want to achieve WhenCluse? I think it is the same as where.

Reply:
To avoid misunderstanding,  we'd better use WhenClause,  of course it's implementation could be the same  with WhereClause.

Reply:
OK, how about  adding the line `using WhenClause = WhereClause;` in src/parser/Clauses.h ?